frontend/README.md

```diff
@@ -213,8 +213,39 @@ test('Application main page is visible', async (t) => {
 
 #### Run all tests locally
 
+E2E-tests are implemented using testcafe (legacy) and playwright.
+Testcafe tests can be run via the following command:
+
 ```sh
 yarn e2e
+```
+
+In order to locally run the playwright tests, some preparation is
+required. Begin by stopping service if it's running:
+
+```sh
+pm2 stop service
+```
+
+Then start the dev server by running the following command in the `service`
+directory:
+
+```sh
+./gradlew bootRunTest
+```
+
```

Comment:
This is actually not required, so you might want to reword this bit.

Reply:
Thanks for the feedback, could you explain this in a bit more detail?

Reply:
I mean that running `service` with `./gradlew bootRunTest` is not required to run playwright tests. It's a useful thing sometimes when you don't want to wipe data from your dev environment, but it's not required.

Reply:
Ok, thanks. Reworded README a bit.